src/Aspire.Dashboard/Components/Controls/SelectResourceOptions.razor

```diff
@@ -0,0 +1,55 @@
+﻿@namespace Aspire.Dashboard.Components
+
+@using System.Collections.Concurrent
+@using Aspire.Dashboard.Model
+@using Aspire.Dashboard.Resources
+
+@inject IStringLocalizer<ControlsStrings> ControlsStringsLoc
+
+@typeparam TValue where TValue : notnull
+
+<FluentStack Orientation="Orientation.Vertical">
+    <FluentCheckbox Label="@ControlsStringsLoc[nameof(ControlsStrings.LabelAll)]"
+                    ThreeState="true"
+                    ShowIndeterminate="false"
+                    ThreeStateOrderUncheckToIntermediate="true"
+                    @bind-CheckState:get="@AreAllVisible"
+                    @bind-CheckState:set="@OnAllValuesCheckedChangedInternalAsync"
+    />
+
+    @foreach (var (resourceType, _) in AllValues
+                  .Where(pair => Resources.Any(resource => ResourceContainsValue(resource, pair.Key)))
```

Comment:
I don't think this can work. The resources here are filtered, so as you filter something out it is no longer available in the list of resources.

IMO keep it simple and display checkboxes for all the values the dashboard has seen.

Reply:
You're right. Reverted